src/app/routes/clientRoutes.js

```diff
@@ -180,3 +180,27 @@ const loadTenantFromLocal = (req, res) => {
     renderDefaultIndexPage(req, res)
   }
 }
+const redirectTologgedInPage = (req, res) => {
+	let redirectRoutes = { '/explore': '/resources', '/explore/1': '/search/Library/1', '/explore-course': '/learn', '/explore-course/1': '/search/Courses/1' };
```

Comment:
why is the first page only considered?
 and why are we hard coding the route /1???